modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/AbstractColocationTest.java

```diff
@@ -68,107 +68,115 @@
 import org.apache.ignite.network.NetworkAddress;
 import org.jetbrains.annotations.Nullable;
 import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.TestInfo;
 import org.junit.jupiter.api.extension.ExtendWith;
 
+// TODO: https://issues.apache.org/jira/browse/IGNITE-22522 remove this test after the switching to zone-based replication
+
+/**
+ * Base class for tests that require a cluster with zone replication.
+ */
 @ExtendWith(ConfigurationExtension.class)
-@ExtendWith(ExecutorServiceExtension.class)
 @ExtendWith(SystemPropertiesExtension.class)
-// TODO: https://issues.apache.org/jira/browse/IGNITE-22522 remove this test after the switching to zone-based replication
+@ExtendWith(ExecutorServiceExtension.class)
 @WithSystemProperty(key = COLOCATION_FEATURE_FLAG, value = "true")
-abstract class ItAbstractColocationTest extends IgniteAbstractTest {
+public class AbstractColocationTest extends IgniteAbstractTest {
```

Comment:
A minor comment: a class is named as `Abstract*` usually means it `abstract`, but the class definition change is  the exact opposite. May be `Base*` prefix will be better?